test/js/api/brand.js

```diff
@@ -28,6 +28,21 @@ describe("BrandAPI", function() {
         });
     });
 
+    describe("#getLogoP()", function() {

```

Comment:
Move this above the `getLogoUrl` test, as you have done for `src/js/api/brand.js`